modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/AutoComplete/index.scss

```diff
@@ -1,10 +1,10 @@
-.dropdown-menu {
+.autocomplete-expand {
 	max-width: 550px;
 	width: 100%;
+}
 
-	&.date-picker-dropdown-menu {
-		max-width: 368px;
-	}
+.dropdown-menu.date-picker-dropdown-menu {
+	max-width: 368px;
 }
 
 @media (max-width: 768px) {
```

Comment:
Hi @Artartur. Thanks for applying the changes. I just think you've missed to change the media query to use the class `autocomplete-expand` instead of `dropdown-menu`:

```
@media (max-width: 768px) {
	.autocomplete-expand {
		max-width: 505px;
	}
}
```
Could you apply it, please? Thanks!

Reply:
my bad, it's fixed now.